core/shortcut_items.ts

```diff
@@ -104,20 +107,18 @@ export function registerCopy() {
         !Gesture.inProgress() &&
         selected != null &&
         selected.isDeletable() &&
-        selected.isMovable()
+        selected.isMovable() &&
+        isCopyable(selected)
       );
     },
     callback(workspace, e) {
-      // Prevent the default copy behavior, which may beep or otherwise indicate
```

Comment:
I think this is actually helpful for context, since we had to add this to solve a specific bug (iirc, mac beeping at you when you used cmd+c). any reason you removed it?

Reply:
Nope will revert!